hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/adapters/IncompatibleMutationAdapter.java

```diff
@@ -115,20 +115,20 @@ private boolean isValidDelete(Cell delete) {
 
         // Incompatible delete: Adapt it.
         try {
-          LOG.debug("Encountered incompatible mutation: " + cell);
+          LOG.info("Encountered incompatible mutation: " + cell);
```

Comment:
should this be warn?

Reply:
This is intentional. 
 
Users will want to know what incompatible mutations did they incur. so by default (Logging level set to info) they will log all the incompatible mutations. 

By setting logger to WARN, they will only log incompatible mutations that are dropped. When they set to ERROR, they will not log any incompatible mutations.